common/src/main/kotlin/com/hartwig/actin/molecular/panel/PanelSpecifications.kt

```diff
@@ -21,13 +21,15 @@ class PanelSpecifications(panelSpecifications: Map<PanelTestSpecification, List<
     ): PanelTargetSpecification {
         val baseTargets = molecularTargetsPerTest[testSpec]
             ?: throw IllegalStateException(
-                "Panel [${testSpec.testName}${testSpec.versionDate?.let { " version $it" } ?: ""}] is not found in panel specifications. Check curation and map to one " +
+                "Panel [${testSpec.testName}${testSpec.versionDate?.let { " version $it" } ?: ""}] " +
+                        "is not found in panel specifications. Check curation and map to one " +
                         "of [${molecularTargetsPerTest.keys.joinToString()}] or add this panel to the specification TSV."
             )
-        val negativeTargets = (negativeResults?.associate { it.gene to listOf(it.molecularTestTarget) } ?: emptyMap())
+        val negativeTargets =
+            (negativeResults?.groupBy(keySelector = { it.gene }, valueTransform = { it.molecularTestTarget }) ?: emptyMap())
```

Comment:
I don't think the named arguments help here:

```suggestion
        val negativeTargets = negativeResults?.groupBy({ it.gene }, { it.molecularTestTarget }).orEmpty()
```